doc/Tutorials.xml

```diff
@@ -181,12 +181,108 @@ There are various examples of packages using &AutoDoc; for only
 this purpose, e.g. <Package>IO</Package> and <Package>orb</Package>.
 <P/>
 
-<!--
-TODO: Explain how to mix AutoDoc content with existing &GAPDoc; XML files.
--->
+<Subsection Label="Tut:IntegrateExisting:EverythingThere">
+  <Heading>Using &AutoDoc; on a complete &GAPDoc; manual</Heading>
+
+Suppose you already have a complete XML manual, with some main and title
+XML files and some documentation for operations distributed over
+all your <F>.g{,d,i}</F> files. Even then &AutoDoc; can do something for you.
+Suppose your main xml file is named <F>PACKAGENAME.xml</F> and is in the
+<F>/doc</F> subfolder of your package. Then calling
```

Comment:
Should be "Then calling the following from a GAP started in the root directory of your package" or so